### ISSUE TITLE
Could we get plants for the Tin/copper/etc nuggets that currently exist in the mod?

### ISSUE THREAD

**Issue by wrincewind** [AgriCraft/AgriPlants#41]
I've been trying to create the resource files for this from scratch but no dice so far - admittedly i've no idea what i'm doing, so i'm flailing somewhat. . .

**Comment by wrincewind** [comment]
I've just about fixed these all up, figured i'd share. 

[Uploading Additional Resource Plants.zip…]()


Note that plombean_plant is not working - it's erroring out for some reason. I think it's breaking because i'm trying to give it a small chance of dropping uranium (makes sense, right?) and it seems that's making it unhappy. 

Here's the offending snippet for Plombean. Can anyone see what I've done wrong?
  "products": {
    "products": [
      {
        "min": 1,
        "max": 2,
        "chance": 0.95,
        "required": true,
        "item": "agricraft:agri_nugget",
        "meta": 6,
        "tags": "",
        "ignoreMeta": false,
        "ignoreTags": [],
        "useOreDict": true
      },
	  {
        "min": 0,
        "max": 1,
        "chance": 0.02,
        "required": false,
        "item": "ic2:resource",
        "meta": 4,
        "tags": "",
        "ignoreMeta": false,
        "ignoreTags": [],
        "useOreDict": false
      }	
    ]

**Comment by wrincewind** [comment]
Leaving the above snippet for posterity, turns out what i did wrong was set the 'min' to 0. I changed min and max to both be 1 and now the output is showing up in JEI. I also lowered the chance to 0.004 to make it a little less common - considering it's an entire block of ore versus one nugget, so... 
Here's the updated, fully working agricraft resource file, with a couple of extra toys in it (namely plants for making alchemical brass, thaumium, quicksilver and ender pearl powder. 
[Agricraft Extra Resources.zip](https://github.com/AgriCraft/AgriPlants/files/2186122/Agricraft.Extra.Resources.zip)
